docs/api-docs/management-apis/contextual-filters.md

```diff
@@ -121,12 +180,30 @@ The response will contain all of the filters available for configuration.
             "price_range_min": 9.95,
             "price_range_max": 225
         },
-        ...
+        {
+            "id": "Q29sb3I=",
+            "type": "product",
+            "name": "Color",
+            "product_count": 1
+        },
+        {
+            "id": "U2l6ZQ==",
+            "type": "product",
+            "name": "Size",
+            "product_count": 1
+        },
+        {
+            "id": "Ym9vbA==",
+            "type": "other",
+            "name": "Other"
+        }
     ],
     "meta": {}
 }
 ```
 
+The filters returned from completing this `GET` request are the filters available for configuration on the given category. In this example, we have Brand, Rating, Price, Color, Size, and Other filters to pick from when choosing which filters to activate for the Kitchen category. In reality, there will be more filters to select from.
```

Comment:
The filters returned from completing this `GET` request are the filters available for configuration on the given category. In this example, we have Brand, Rating, Price, Color, Size, and Other filters to pick from when choosing which filters to activate for the Kitchen category. In reality, you will be able to select from more filters.